awx_collection/README.md

```diff
@@ -71,6 +71,30 @@ pip install -e .
 PYTHONPATH=awx_collection:$PYTHONPATH py.test awx_collection/test/awx/
 ```
 
+## Running Integration tests Tests
+
+The integration tests require a virtualenv with `ansible` >= 2.9 and `tower_cli`.
+The collection must first be installed, which can be done using `make install_collection`.
+You also need a configuration file at `~/.tower_cli.cfg` or
+`/etc/tower/tower_cli.cfg` with the credentials for accessing tower. This can
+be populated using `tower-cli`:
+
+```
+tower-cli config host $HOST
+tower-cli config username $USERNAME
+tower-cli config password $PASSWORD
+# This tells the tower-cli not to veriffy the ssl certs in the tower, if your tower has good certs you should leave this to true
+tower-cli config verify\_ssl false
+```
+
+Finally you can run the tests:
+
+```
+# ansible-test must be run from the directory in which the collection is installed
+cd ~/.ansible/collections/ansible\_collections/awx/awx/
```

Comment:
I don't think the backslash at "ansible\_collections" is necessary

Reply:
I second @AlanCoding's comment.